packages/web3/package.json

```diff
@@ -30,16 +30,18 @@
 		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
 		"test:watch": "npm test -- --watch",
 		"test:unit": "jest --config=./test/unit/jest.config.js",
-		"test:integration": "jest --config=./test/integration/jest.config.js --passWithNoTests"
+		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --detectOpenHandles --forceExit"
 	},
 	"devDependencies": {
+		"@truffle/hdwallet-provider": "^2.0.4",
 		"@types/jest": "^27.0.3",
 		"@typescript-eslint/eslint-plugin": "^5.4.0",
 		"@typescript-eslint/parser": "^5.4.0",
 		"eslint": "^8.3.0",
 		"eslint-config-prettier": "^8.3.0",
 		"eslint-config-web3-base": "0.1.0",
 		"eslint-plugin-import": "^2.25.3",
+		"isomorphic-ws": "^4.0.1",
```

Comment:
I'm confused why this is needed for this package, it doesn't seem to be used in these changes (assuming the reason is related to `tools/isomorphic-ws.js`)?

Reply:
You are right, I copied it from another branch and I thought it was required, but, it's not. I'll remove it. @avkos could you please verify it's not needed?

Reply:
Yes it's not required as dependency in this package.